integrations/os2sync/os2mo.py

```diff
@@ -198,6 +200,29 @@ def addresses_to_orgunit(orgunit, addresses):
             orgunit["Post"] = a["value"]
 
 
+def kle_to_orgunit(orgunit, kle):
+    """
+    Aspect Udførende goes into Tasks
+    Aspect Ansvarlig goes into ContactForTasks
+    """
```

Comment:
Her kunne jeg godt drømme om input / output dokumentation.
Jeg tænker:
```suggestion
def kle_to_orgunit(orgunit, kle):
    """Collect kle uuids according to kle_aspect.
    
    * Aspect "Udførende" goes into "Tasks"
    * Aspect "Ansvarlig" goes into "ContactForTasks"
    
    Example:
    
        kles = [
            {'uuid': 1, 'kle_aspect': [{'scope': 'UDFOERENDE'}]},
            {'uuid': 2, 'kle_aspect': [{'scope': 'ANSVARLIG'}]},
            {'uuid': 3, 'kle_aspect': [{'scope': 'ANSVARLIG'}, {'scope': 'UDFOERENDE'}]}
        ]
        kle_to_orgunit(orgunit, kles)
        self.assertEqual(orgunit['Tasks'], [1, 3])
        self.assertEqual(orgunit['ContactForTasks'], [2, 3])
    
    Args:
        orgunit: The organization unit to enrich with kle information.
        kle: A list of KLEs.
    
    Returns:
        None
    """
```
Det giver samtidig en test for metoden.

Reply:
Samler du doctests op i koden?
Vi har ikke brugt doctests før i os2mo-data-import-and-export, mener jeg, men den der er da til at have med at gøre.

Reply:
Nej, min pointe var blot at man kunne have flyttet det ud i en test fil nemt.

Reply:
Jeg synes tit at doctests bliver besværlige, ifht. at man ikke kan skrive alle sine tests der, så man skal alligevel have en test fil, og hvad var så pointen.